torchlpc/core.py

```diff
@@ -159,7 +161,12 @@ class LPC(Function):
     def forward(x: torch.Tensor, A: torch.Tensor, zi: torch.Tensor) -> torch.Tensor:
         if x.is_cuda:
             y = lpc_cuda(x.detach(), A.detach(), zi.detach())
+        elif EXTENSION_LOADED:
+            y = torch.ops.torchlpc.lpc_cpu(x, A, zi)
```

Comment:
Just curious, why do you call `.detach()` for GPU but not for CPU?

Reply:
To properly expose the underneath storage to the other framework, at least this is the case for numba. For Torch C++ API, this is not necessary, as they accept Tensor type.